setup.cfg

```diff
@@ -38,7 +38,7 @@ install_requires =
   pyarrow
   seaborn
   opencv-python
-
+  click
```

Comment:
What does this do? [sorry for my ignorance]

Reply:
Just a module that allows defining of hyper parameters when running files from the terminal!